worker/src/DepLibUring.cpp

```diff
@@ -30,37 +31,83 @@ inline static void onFdEvent(uv_poll_t* handle, int status, int events)
 	// the counter in order to avoid libuv calling this callback indefinitely.
 	eventfd_t v;
 	int err = eventfd_read(liburing->GetEventFd(), std::addressof(v));
+
 	if (err < 0)
 	{
-		MS_ABORT("eventfd_read() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
```

Comment:
I don't think `-1 *` is needed.

Reply:
What? You mean uysing `-err` instead or what?

Reply:
If so feel free to commit and merge at your convenience, I'll go out in a few minutes.

Reply:
Yes, you can simply negate `err`. `-1 *` was an attempt you did the fix the compiler error related to using `errno` local variable.

Reply:
You mean this?

```c++
err -= err;
```

I just don't want to pass `-err` to functions.

Reply:
I see your commit now. Ok.